lib/vm/src/instance/ref.rs

```diff
@@ -156,7 +156,7 @@ impl Clone for InstanceRef {
         Self {
             strong: self.strong.clone(),
             instance_layout: self.instance_layout,
-            instance: self.instance.clone(),
+            instance: self.instance,
```

Comment:
`self.instance` is of type `NonNull<T>`. It implements `Copy`, it's then useless to clone it.